pos_sale_color_code_tintometric_system/models/stock_move.py

```diff
@@ -0,0 +1,30 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import models, fields, api
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    move_pos_order_line_id = fields.Many2many("pos.order.line")
```

Comment:
Due to usual `Many2many` conventions, rename this:
```suggestion
    move_pos_order_line_ids = fields.Many2many("pos.order.line")
```